charts/node/README.md

```diff
@@ -96,6 +96,10 @@ node:
 
 ## Upgrade
 
+### From v5.x.x to v5.3.0 (⚠️ breaking changes)
```

Comment:
Make sure to state that those are breaking changes in an experimental option only. Otherwise this would require a major version bump. As it is, it's unlikely to affect anyone besides us.

Reply:
Yes. I added two of the critical information. 

The version from which is affected starting from 5, and the variable that was changed.